taxi/tests.py

```diff
@@ -1,3 +1,128 @@
+from django.contrib.auth import get_user_model
+from django.urls import reverse
 
-# Create your tests here.
+
+class ModelsTests(TestCase):
+    def test_driver_str(self):
+        driver = get_user_model().objects.create(
+            username="leleka",
+            password="password",
+            first_name="Ivan",
+            last_name="Olevsky",
+            email="ivan@example.com",
+            license_number="AB123456789"
+        )
+        self.assertEqual(str(driver),
+                         f"{driver.username} ({driver.first_name} {driver.last_name})")
+
+
+
+    def test_driver_with_license_number(self):
+        password = "TEST PASSWORD"
+        username = "Alexov"
+        first_name = "Alex"
+        last_name = "Statham"
+        email = "alex@example.com"
+        license_number = "1234567890"
+
+        driver = Driver.objects.create_user(
+            username=username,
+            password=password,
+            first_name=first_name,
+            last_name=last_name,
+            email=email,
+            license_number=license_number,
+        )
+        self.assertEqual(str(driver.license_number), license_number)
+        self.assertTrue(driver.check_password(password))
+        self.assertEqual(driver.get_full_name(), f"{first_name} {last_name}")
+
+    def test_manufacturer_str(self):
+        name = "Ivan"
+        country = "Ukraine"
+
+        manufacturer = Manufacturer.objects.create(name=name, country=country)
+        self.assertEqual(str(manufacturer), f"{manufacturer.name} {manufacturer.country}")
+
+
+class AdminTestCase(TestCase):
+    def setUp(self):
+        self.client = Client()
+        self.admin_user = get_user_model().objects.create_superuser(
+            username="admin",
+            password="password1",
+        )
+        self.client.force_login(self.admin_user)
+        self.driver = get_user_model().objects.create_superuser(
+            username="Karina",
+            password="password",
+            license_number="tes01234"
+        )
+
+    def test_admin_listed(self):
+        url = reverse("admin:taxi_driver_changelist")
+        res = self.client.get(url)
+        self.assertContains(res, self.driver.username)
+
+    def test_admin_listed_detail(self):
+        url = reverse("admin:taxi_driver_change", args=[self.driver.id])
+        res = self.client.get(url)
+        self.assertContains(res, self.driver.username)
+
+
+class FormTestCase(TestCase):
+    def test_driver_creation_form_valid(self):
+        form_data = {
+            "username": "test_user",
+            "password1": "StrongP@ssw0rd!",
+            "password2": "StrongP@ssw0rd!",
+            "first_name": "Test",
+            "last_name": "User",
+            "email": "test_user@example.com",
+            "license_number": "ABC12345",
+        }
+        form = DriverCreationForm(data=form_data)
+        self.assertTrue(form.is_valid(), msg=f"Form errors: {form.errors}")
```

Comment:
It's not a good practice to print anything in your tests. If you want to check the form's cleaned data, you should write an assertion for it.